docs/applications/integrate-application-deploy-to-source-provider-ci.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Integrate Application Deploy to Source Provider's CI
```

Comment:
Outgoing hooks are not limited to integrate with source providers. Devopness can integrate with any CI, CD, CI/CD workflows and even beyond that:
```suggestion
title: Integrate Application deployments with your Continuous integration (CI) workflows
```